### DISCUSSION TITLE
articles/naval

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#182] [Comments]
# articles/naval

纳瓦尔·拉维坎特(Naval Ravikant)是美国硅谷的风险投资家，他出生在印度，儿时随父母搬到纽约，家境贫穷，靠着自己的努力一步步走上财富自由之路，创办了AngelList风险投资公司，成功投资了Twitter、Uber等初创公司。纳瓦尔经常在博客、推特、播客上分享他的人生哲学，最著名的是他在推特上名为《如何不靠运气致富》的系列推文。而记者埃里克·乔根森将他在网络上的发言集结成书，也就是这本《纳瓦尔宝典》。去年我大略读了一遍，有些感触，但并不深刻。今年在经历了极度的焦虑抑郁状态后，重读此书，突然有一种醍醐灌顶的感觉，于是我又在有声书上听了数遍，细心体会纳瓦尔的人生哲学。

https://hutusi.com/articles/naval

**Comment by vra** [comment]
感谢分享，“人生的意义在于做局部熵减”的观点让我很受启发

**Reply by wangyunzi** [reply]
同样很有感悟。最近感觉我的下半辈子都要一直致力于做这件事了✌️

**Comment by NOeSIS-yu** [comment]
被安利了

**Comment by linlinzzo** [comment]
打工是不可能致富的，哎。

**Comment by iyuanze** [comment]
对于第三种的人生的意义是不是：人类的所有行为都是在帮助宇宙加速熵增，达到热寂，这是不可逆的。但是对于个人来说，可以通过局部的熵减，来追求自己的幸福？
- [涓滴意念汇成河](https://www.zahui.top/)

**Comment by Newdea** [comment]
看前面以为看到了老子的”知足者富“，但后面又跟有叔本华的幸福的无聊。唉，也许无论何时，我们都只能在路上，先是求财之路上，知足后就转到求幸福的路上，但幸福是一种内心状态，是无法通过现实行为明确达到的，是要通过悟达到这样种状态，或说境界。也许在经历些事后，不经意间就明白了；也许终尽一生，还觉得不够幸福。
记得陈嘉映老师曾说过：良好的生活，就是行之于途而应于心。行为即是目的、意义。当我们能直接从做事中感到喜乐幸福时，就足够了。

**Reply by hutusi** [reply]
谢谢，你的观点很有启发。“行之于途而应于心”，和《金刚经》中的“应无所住而生其心” 有异曲同工之妙~~

**Comment by elthon** [comment]
谢谢楼主的分享，看完很有感触，感觉现在人不像之前是浮躁的状态，更多了焦虑的状态。 可能和年龄有关系也可能和环境有关系，我觉得还是缺少目标导致了焦虑，如果我们处在追逐自己的梦想和目标的路上，获得财富和幸福应该是水到渠成的事情。

**Reply by hutusi** [reply]
按照存在主义的观点，人是必然存在焦虑的~~

**Comment by Newdea** [comment]
嗯，就是侧重点不同，应无所住而生其心，是侧重不着相。行之于途而应于心，侧重行动，随心而动



---Original---
From: "John ***@***.***&gt;
Date: Sun, Jan 14, 2024 20:25 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hutusi/hutusi.github.com] articles/naval (Discussion #182)




 
谢谢，你的观点很有启发。“行之于途而应于心”，和《金刚经》中的“应无所住而生其心” 有异曲同工之妙~~

**Comment by WeiHuangWH** [comment]
在如此快节奏的时代，自己浮躁且肤浅的心随着时代浪潮恰如其分地浮沉了好久。
机缘巧合翻开作者的博客（这应该是一个极其小众到近乎于上世纪的交流方式），找到了让自己慢下来保持镇静和坚定的力量。或许是智慧的美好，也或许是来自某种同类的感召。缘分，真是奇妙。
希望能在人生接下来的道路中，用更符合自己期待的方式奋力前进，富足且幸福着。加油！

**Reply by hutusi** [reply]
谢谢，加油！
